File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/upload_init_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class UploadInitForm:
    """上传初始化请求"""
    file_name: str = None
    file_size: int = None
    file_md5: str = None
    file_type: str = None
    upload_type: str = None
    chunk_size: int = None
    total_chunks: int = None
    storage_path: str = None
    business_type: str = None
    business_id: str = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/UploadInitForm.kt
package com.sdkwork.app

data class UploadInitForm(
    val fileName: String? = null,
    val fileSize: Int? = null,
    val fileMd5: String? = null,
    val fileType: String? = null,
    val uploadType: String? = null,
    val chunkSize: Int? = null,
    val totalChunks: Int? = null,
    val storagePath: String? = null,
    val businessType: String? = null,
    val businessId: String? = null
)
